src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -132,7 +128,11 @@ public void exec(final URI home) throws IOException {
      * BkBasic produces headers with addresses without slashes.
      *
      * @throws IOException If some problem inside
+     * @todo #519:30min In org.takes.rq.RqHeaders.Base#map is skipping the
```

Comment:
@carlosmiranda I think, I find the problem. 
This is the content we have in the test socket
```
                Joiner.on(BkBasicTest.CRLF).join(
                    "GET / HTTP/1.1",
                    "Host:localhost",
                    "Content-Length: 2",
                    "",
                    "hi"
                ).getBytes()
```

In [RqLive:parse line 99](https://github.com/yegor256/takes/blob/master/src/main/java/org/takes/rq/RqLive.java#L99)
we are breaking the loop. so "hi" remains in the stream. So in the next iteration of BkBasic accept code I have added, try to create the header from just "hi". So the test breaks.

Hope you understood my explanation. Can I add this information too in the puzzle? or should I create a new issue as the `RqLive#parse` is not completely parsing the request. This problem only show up when we want the connection keep alive.